CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 * [CHANGE] Ruler: Remove unused CLI flags `-ruler.search-pending-for` and `-ruler.flush-period` (and their respective YAML config options). #2288
 * [CHANGE] Successful gRPC requests are no longer logged (only affects internal API calls). #2309
 * [CHANGE] Add new `-*.consul.cas-retry-delay` flags. They have a default value of `1s`, while previously there was no delay between retries. #2309
+* [FEATURE] Compactor: Adds the ability to delete partial blocks after a configurable delay. This option can be configured per tenant. #2285
+  - `compactor.partial-block-deletion-delay`, as a duration string, allows you to set the delay since a partial block has been modified before marking it for deletion. A value of `0s`, the default, disables this feature.
+  - `cortex_compactor_blocks_marked_for_deletion_total` has a new label `reason="partial"`, when a deletion marker is triggered by the partial block deletion delay.
```

Comment:
I think it helps disambiguate to clarify that this is a metric, at first I thought it was a configuration parameter:
```suggestion
  - The metric `cortex_compactor_blocks_marked_for_deletion_total` has a new label `reason="partial"`, when a block deletion marker is triggered by the partial block deletion delay.
```

Reply:
`reason` is not a new label, but `partial` is a new label value for that label.